src/Common/Constants.ts

```diff
@@ -13,6 +13,8 @@ export default class Constants {
   public static readonly defaultLocale = "en-us";
   public static readonly noContentStatusCode = 204;
   public static readonly tooManyRequestsStatusCode = 429;
+  public static readonly badRequest = 429;
```

Comment:
Isn't `bad request` supposed to be `400`? 

Should we rename it to `badRequestStatusCode` to keep namings consistent?